src/store.ts

```diff
@@ -29,7 +27,9 @@ import { DbUserStore } from "./db/userstore";
 import {
     RoomStore, UserStore,
 } from "matrix-appservice-bridge";
+
 const log = new Log("DiscordStore");
+export const CURRENT_SCHEMA = 10;
```

Comment:
why the location switch?

Reply:
I was annoyed that we had the schema in the middle of our imports